src/applications/disability-benefits/781/pages/ptsdChoice.js

```diff
@@ -1,70 +1,34 @@
-import React from 'react';
-import {
-  ptsdNameTitle,
-} from '../helpers';
-import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+import { ptsdNameTitle781, uploadPtsdDescription781, ptsdChoiceDescription } from '../helpers';
 
-const uploadPtsdDescription = () => {
```

Comment:
This content was left in this file because it seemed specific to this file/functionality. The true effort was to keep the helpers.js file as slim as possible. If you feel this content is reusable, then I'm totally with it going to the helpers file.

Reply:
@igreulich  suggested that all helpers be kept in helpers. So I moved it to helpers. You guys let me know what's best, and I will do it. Thanks! @robfelder

Reply:
@imeecuison @robfelder All the reusable stuff should go in helpers. All the once and done stuff can go in the file in the pages dir.